src/Common/test/Common.Expression.Test/Spring/SpelCompilationCoverageTests.cs

```diff
@@ -5756,22 +5751,6 @@ private string Stringify(object obj)
                 s.Append(' ');
             }
         }
-        else if (obj is object[] os)
```

Comment:
Can we trust this static analysis? Or do we need to improve input/test coverage to validate this?

Reply:
I'm confident this is correct. `System.Array` implements `System.Collections.IList`, see [here](https://learn.microsoft.com/en-us/dotnet/api/system.array), so the removed code is never hit. But more importantly, even if it were, it would do the exact same thing. In another place, the logic for the branches was different, so I moved things around because it looked like a bug.